components/dashboard/learning-path-item.tsx

```
@@ -78,29 +78,29 @@ export function LearningPathItem({
 			</div>
 
 			{expanded && (
-				<div className="mt-4 space-y-2 rounded-md border border-gray-800 bg-gray-950 p-3">
+				<div className="mt-4 space-y-2 rounded-md border border-gray-200 dark:border-gray-800 bg-gray-50 dark:bg-gray-950 p-3">
 					{modules.map((module, index) => (
 						<div key={index} className="flex items-center justify-between">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)